portality/scripts/api_export.py

```diff
@@ -0,0 +1,135 @@
+import esprit, os, shutil, tarfile, json, codecs
+from portality.core import app
+from portality.lib import dates
+from portality.store import StoreFactory
+from portality.api.v1.common import ModelJsonEncoder
+from portality.api.v1 import DiscoveryApi, DiscoveryException, jsonify_models
+from urlparse import parse_qs, urlparse
+from datetime import datetime
+
+def _save_file(typ):
+    # Create a dir for today and save all files in there
+    data = json.dumps(results, cls=ModelJsonEncoder)
+    filename = os.path.join(today, "{typ}_{page}.json".format(typ=typ, page=page))
+    output_file = tmpStore.path(container, filename, create_container=True, must_exist=False)
+    dir = os.path.dirname(output_file)
+    if not os.path.exists(dir):
+        os.makedirs(dir)
+    print("Saving file {filename}".format(filename=filename))
+    out = codecs.open(output_file, "wb", "utf-8")
+    out.write(data)
+    out.close
+    return
+
+
+def _get_last_page():
+    url = results.data.get('last', '')
+    last_page = None
+    if url:
+        last_page = parse_qs(urlparse(url).query).get('page', [])
+    if last_page:
+        return int(last_page[0])
+    else:
+        return None
+
+def _get_dir_size(path):
+    total_size = 0
+    for dirpath, dirnames, filenames in os.walk(start_path):
```

Comment:
```suggestion
    for dirpath, dirnames, filenames in os.walk(path):
```
no variable `start_path`